app/views/layouts/angular.html.haml

```diff
@@ -20,11 +20,72 @@
     = render 'angular/google_analytics'
     = render 'angular/intercom' if current_user_or_visitor.is_logged_in?
 
-  %body{'ng-keydown' => 'keyDown($event)'}
+  %body{'ng-keydown' => 'keyDown($event)', layout: "column", flex: ""}
     %flash
     %navbar
-    %ng_outlet{'ng-if' => '!refreshing && !pageError'}
-    %error_page{'ng-if' => 'pageError', 'error' => 'pageError'}
+    %md_content.lmo-sidebar-and-main-container{layout: "row", flex: ""}
```

Comment:
OoOoh... I guess there was a thing about the sidebar not behaving well in a component.